resources/benchmark-runner.mjs

```diff
@@ -432,10 +428,13 @@ export class BenchmarkRunner {
         const suites = await this._prepareAllSuites();
         try {
             for (const suite of suites) {
-                if (!suite.disabled)
+                if (!suite.disabled) {
+                    await this._appendFrame();
+                    this._page = new Page(this._frame);
                     await this.runSuite(suite);
+                    this._removeFrame();
```

Comment:
nit: please put a try/finally around `appendFrame`, `new Page` and `runSuite` and call `removeFrame` in the `finally` block.

Reply:
.. and we can save an indent by doing
```
if (suite.disabled)
  continue;
```
instead.